### PR TITLE
Diary Fixes

### PR DESCRIPTION
**Pull request by Drapegnik**
Screen Shot 2019-09-26 at 13 56 01

### github deployments
Screen Shot 2019-09-26 at 13 55 42
Screen Shot 2019-09-26 at 14 30 12

### CODE REVIEW

**Comment by Drapegnik** [comment]
[Diary Fixes](https://trello.com/c/ZsQ7Z16J/408-diary-fixes)

File: components/articles/blocks/diary.scss [hunk 1]
```diff
@@ -99,7 +111,7 @@
 
   .diary {
     &__text {
-      width: 570px;
+      // width: 570px;
```
Comment (UladBohdan) [inline components/articles/blocks/diary.scss:114]: do we need to keep these commented lines?

**Review by UladBohdan — DISMISSED** [review]
probably to implement modal diary window in this PR?..

File: .travis.yml [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
 after_success:
-- npm run travis-deploy-once "npm run semantic-release"
+  - npm run travis-deploy-once "npm run semantic-release"
 notifications:
   slack:
     rooms:
       secure: IJQ/In8oX+KAdJ4REPp81F8KIa/LTymLmWRdOLFSv06Ju+hOa69vi0DmTKgd9nYy+vzxMsJDCSStIJ+knKtfk5B6OZYNAmq/Etz388Ly2Ih/S+06XAMLJf3/uc61eLpPjmFf6fImZo+PRhPlNW21U9AEXRG0COga2T8AAH8dWPuZjbnK/kY5JyjtzuQti5/GW+M7YiYxfw7UKIiIfBxZXNYbeE1l+R9LVIaw2pwbVtDTpw61JosTTWmq/fvOsXCGYHLigJmLLLliK9f35aij4Wa612RFMSBZ0FaYbp6Flt5RlbTOuwe01u4CuAVMfc4h4DFoxCOhIU7R7ZFqX9oOTVl7Z2lVvvOYloHMzgKG20g4aV0pqVxX+yl3lh4dMQ6mtzTIaI75w9MIEzg2sxx6lMrO76P576W0L6VXs5Qt29/9MHf8CQ8MowpxTChncH6fBD66ebHX+Ixo3yTyjsDiXG9LkwFmMjE7k4/BqYcwjwO5Gb4Wn1IBWTE1iL/Nq4kIUsQ+eA2LgwxP1F/ARQZO0N8ZXfSlSiUjn0ouF4CrFgq5ZP1obT4fIi7+cJC8I54K6x8O3hcSEtdT5tHV3QettDYki/d3Eh3SuIDP5gVciLm1x+5txWrIM6zjU1AO/e8AV5SSkHSURgiv3wphw8nQUxSC4DkEknPsGPVCPbGbSVY=
 addons:
   ssh_known_hosts: dev.wir.by
 before_deploy:
```
Comment (UladBohdan) [inline .travis.yml:16]: we will later need to make separate `before_deploy` for dev and prod. Now it is dev-specific. Is there a way to only make it run on push to `develop` branch?

Comment (UladBohdan) [reply .travis.yml:16]: same for `after_deploy` script.

**Review by UladBohdan — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (Drapegnik) [reply .travis.yml:16]: `sh bin/dev/after-deploy.sh dev` - it already receive `env` param

**Review by UladBohdan — APPROVED** [review]
Verified status is pushed properly when deploying from local.

**Pull request merged**